src/array/list/mod.rs

```diff
@@ -129,19 +129,14 @@ impl<O: Offset> ListArray<O> {
     /// Returns the element at index `i`
     #[inline]
     pub fn value(&self, i: usize) -> Box<dyn Array> {
-        if self.is_null(i) {
```

Comment:
Is this fine that you're no longer creating an empty list child?

Reply:
this was unrelated. The behavior of all others is that `value` returns the values irrespectively of the validity buffer. This keeps that behavior consistent.